### PR TITLE
Update Bulletin promo to match desktop layout

### LINKED ISSUE #3017: Update Bulletin Promo layout
**Is your feature request related to a problem? Please describe.**
Currently the Bulletin Promo in the desktop layout (1008px and above) still has the single layout
Currently:
Screenshot of current layout
Desired design:
Screenshot of designs

For the desktop layout, we need to update the promo so it matches designs. 

**Describe the solution you'd like**
Update styles within the bulletin-promo for 1008px+. 

**Describe alternatives you've considered**
We've considered exporting the parts of the Bulletin promo separately and handling layout in StoryPromoContainer in Simorgh.  This may be useful for keeping promos in sync as we maintain the codebase. The approach above is faster. 

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #3017

**Overall change:** 
Update Bulletin styles to match 1008+px layout design

**Code changes:**
- Refactor Bulletin to follow the same approach as the `StoryPromo`
- Move `Image` and `Text` styles to different files
- Apply different media queries to the Radio and TV bulletins to match the tablet and desktop designs
- Apply `Pica` to Radio headline and `GreatPrimer` to TV headline
- Add `ImageGridItem` and `TextGridItem` which accept a `bulletinType` prop to get either the `radio` or `tv` styles
- Pass `bulletinType` prop also to `BulletinHeading` and `BulletinSummary` rather than having two different components with the Radio and TV styles for each of them
- Do not render the `ImageGridItem` if the bulletin promo doesn't have an image and tweak the Radio `TextGridItem` styles to take the full width on tablet
- Add `rtl` support to `BulletinSummary` to apply the right paddings
- Update snapshots

TV bulletin has a layout change at 600px. Radio bulletin has a layout change at 600px and 1008px:

**TV Bulletin below 600px:** 
Screenshot TV bulletin <= 599px

**TV Bulletin 600px+:**
Screenshot TV bulletin 600px+

**Radio Bulletin below 600px:**
Screenshot Radio bulletin below 600px

**Radio Bulletin 600px-1007px:**
Screenshot Radio bulletin 600px-1007px

Without image:

radio-tablet-no-image

**Radio Bulletin 1008px+:**
Screenshot Radio bulletin above 1008px

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [X] This PR requires manual testing

### CODE REVIEW

**Review by AlistairGempf — APPROVED** [review]
Looks good to me

**Review by sareh — APPROVED** [review]
👍 Looks good. Thanks for the update. 

At 1008px and above, we get the layout with the image over the text.  Screenshot bulletin promo
http://localhost:8180/?path=/story/components-bulletin--tv-bulletin

**Review by OlgaLyubin — APPROVED** [review]
LGTM 👍

**Review by sareh — CHANGES_REQUESTED** [review]
On the IE11 snapshots, we get the image is not spanning the full width of the bulletin promo. We should ensure that it does span the full width. 
Screenshot IE11
https://www.chromaticqa.com/snapshot?appId=5d28eb5ee163f6002046d6fb&id=5e2ef367d5d60a0022bc1bc4

File: packages/components/psammead-bulletin/src/ImageStyles.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+const imageGridFallbackStyles = {
+  radio: imageGridFallbackRadio,
+  tv: imageGridFallbackTv,
+};
+
+const ImageGridItem = styled.div`
+  vertical-align: top;
+  display: inline-block;  
+  width: ${fullWidthColumnsMaxScaleable};
+  padding: ${GEL_SPACING} ${GEL_SPACING} 0 ${GEL_SPACING};
```
Comment (AlistairGempf) [inline packages/components/psammead-bulletin/src/ImageStyles.jsx:55]: I think the radio bulletin image should have no padding on desktop.
image

Comment (DenisHdz) [reply packages/components/psammead-bulletin/src/ImageStyles.jsx:55]: Missed that one :)

File: packages/components/psammead-bulletin/src/index.jsx [hunk 2]
```diff
@@ -67,52 +45,86 @@ const TVBulletinWrapper = styled.div`
...
+  `;
+};
+
+const headingStyles = css`
+  ${({ service }) => service && getSerifMedium(service)}
+  ${({ script }) => script && getPica(script)}
+  color: ${C_EBON};
+  margin: 0; /* Reset */
+  padding: ${GEL_SPACING};
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
```
Comment (sareh) [inline packages/components/psammead-bulletin/src/index.jsx]: We will need to add another media query for groupB.
If you update the `headlineTypography` function to also take in a group string, e.g. 'groupD', we could have this:

```
@media (min-width: ${GEL_GROUP_B_MIN_WIDTH}) and (max-width: ${GEL_GROUP_B_MAX_WIDTH}) {
  ${({ script }) => script && headlineTypography(script, 'groupB')}
}
@media (min-width: ${GEL_GROUP_D_MIN_WIDTH}) {
  ${({ script }) => script && headlineTypography(script, 'groupD')}
}

Comment (sareh) [reply packages/components/psammead-bulletin/src/index.jsx]: I checked on browserstack and it is all working well.

**Review by sareh — APPROVED** [review]
👍 Great work on this PR! Looks good.

File: packages/components/psammead-bulletin/src/index.jsx [hunk 3]
```diff
@@ -15,22 +18,16 @@ import {
   getSerifMedium,
 } from '@bbc/psammead-styles/font-styles';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
-import { getPica, getLongPrimer } from '@bbc/gel-foundations/typography';
+import {
+  getPica,
+  getGreatPrimer,
+  getLongPrimer,
```
Comment (sareh) [inline packages/components/psammead-bulletin/src/index.jsx:24]: 👍 Updates to typography settings look good! 
The summary typography matches that of StoryPromo's Summary. 
The TV bulletin headline typography matches the StoryPromo Leading heading.
The Radio bulletin headline typography matches the StoryPromo regular heading.

**Review by sareh — APPROVED** [review]

**Review by AlistairGempf — APPROVED** [review]
Good job 👍

**Comment by staylos92** [comment]
Looks good to me 👍 

Confirmed the image is the correct size in IE11 too

**Pull request merged**